pkg/controller/unifiedpushserver/postgresql.go

```diff
@@ -181,3 +181,27 @@ func newPostgresqlService(cr *pushv1alpha1.UnifiedPushServer) (*corev1.Service,
 		},
 	}, nil
 }
+
+func newPostgresImageStream(cr *pushv1alpha1.UnifiedPushServer) (*imagev1.ImageStream, error) {
+	return &imagev1.ImageStream{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: cr.Namespace,
+			Name:      cfg.PostgresImageStreamName,
+			Labels:    labels(cr, cfg.PostgresImageStreamName),
+		},
+		Spec: imagev1.ImageStreamSpec{
+			Tags: []imagev1.TagReference{
+				{
+					Name: cfg.PostgresImageStreamTag,
+					From: &corev1.ObjectReference{
+						Kind: "DockerImage",
+						Name: cfg.PostgresImageStreamInitialImage,
+					},
+					ImportPolicy: imagev1.TagImportPolicy{
+						Scheduled: true,
```

Comment:
According to @maleck13, we shouldn't set this scheduled part (for now at least). Image imports should instead be done manually during a scheduled maintenance window.

Reply:
Ok, will apply that to all image streams we create here